src/components/Cart/Summary/Buttons.vue

```diff
@@ -38,26 +38,26 @@ export default {
   setup(props: any) {
     const store = useStore();
     const formValid: ComputedRef<boolean> = computed(() => props.isFormValid);
-    const stepValue: ComputedRef<number> = computed(() => store.getters['sideMenu/stepValue']);
-    const price: ComputedRef<number> = computed(() => store.getters['cart/priceTotal']);
+    const stepValue: ComputedRef<number> = computed(() => store.getters["sideMenu/stepValue"]);
+    const price: ComputedRef<number> = computed(() => store.getters["cart/priceTotal"]);
 
-    const isChecked: Ref<()=>boolean> = ref((): boolean => {
-      return (!formValid.value && stepValue.value === 2 || price.value === 0 && stepValue.value === 1)
-    })
+    const isChecked: Ref<() => boolean> = ref((): boolean => {
```

Comment:
i suggest using `computed` instead of `ref` if you want to watch dependencies inside of a function

Reply:
Computed does not work on this case, when i changed that the whole cart has crashed. It is working well with just ref. All the values inside of ref (formValid.value , setpValue.value or price.value) are computed themselves.